bigquery/google/cloud/bigquery/_helpers.py

```diff
@@ -318,7 +318,7 @@ def _validate(self, value):
         :raises: ValueError if value is not allowed.
         """
         if value not in self.ALLOWED:
-            raise ValueError('Pass one of: %s' ', '.join(self.ALLOWED))
```

Comment:
No worries here, but a point of note when using percent (`%`) formatting in Python. To be extra super-duper safe, you should always pass a tuple:

```python
>>> format = 'see %s'
>>> value = 'this'
>>> format % value
'see this'
>>> value = ('a', 'real', 'tuple', 9)
>>> format % value
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: not all arguments converted during string formatting
>>> format % (value,)
"see ('a', 'real', 'tuple', 9)"
```

In this case it's obvious we aren't accidentally passing a tuple as the single argument